emission/__TEST_0_INIT_VARIABLE_AND_HELPER_test.gno

```diff
@@ -0,0 +1,64 @@
+package emission
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/r/demo/gns"
+
+	"gno.land/r/demo/gnoswap/consts"
+
+	pusers "gno.land/p/demo/users"
+)
+
+var (
+	gsa std.Address = consts.GNOSWAP_ADMIN
+)
+
+// Realms to mock frames
+var (
+	gsaRealm = std.NewUserRealm(gsa)
+
+	govRealm = std.NewCodeRealm(consts.GOV_PATH)
+)
+
+/* HELPER */
```

Comment:
Please adhere to Golang's comment conventions.
// gnsBalance ......

Reply:
I don't think we need to write comments for test related functions

Reply:
33592c821dfd47060eed3af81baa6009e69a7db2